src/_assets/css/_dash.scss

```diff
@@ -6,7 +6,7 @@ $dash-dark-black: #4a4a4a;
 $dash-highlight: #1967d2;
 $dash-off-black: #12202f;
 $dark-darker-black: #0d1520;
-$dash-header-bg: #17212f;
+$dash-header-bg: #12202f;
```

Comment:
On macOS the background color of the image seems to be #17212f. This change causes a slight issue with the background:

<img width="611" alt="Screen Shot 2019-05-06 at 10 51 09 AM" src="https://user-images.githubusercontent.com/1145719/57244279-e9fabf00-6fec-11e9-926e-da79193bf559.png">

Reply:
@johnpryan is this still an issue?

Reply:
yeah, I can make a PR to master

Reply:
PR here: https://github.com/dart-lang/site-www/pull/1581